lib/active_fedora/attributes.rb

```diff
@@ -9,7 +9,26 @@ module Attributes
       include Serializers
       include PrimaryKey
 
-      after_save :changes_applied
+      after_save :clear_changed_attributes_or_changes_applied
+      def clear_changed_attributes
+        Deprecation.warn ActiveFedora::Attributes, "#clear_changed_attributes is deprecated, use ActiveModel::Dirty#changes_applied instead."
+        @previously_changed = changes
+        clear_attribute_changes(changes.keys)
+      end
+
+      ##
+      # Call the deprecated method
+      def clear_changed_attributes_or_changes_applied
```

Comment:
Is the plan to remove this method in AF 14?  If so, do we need a comment saying that it too is deprecated.

Reply:
maybe we should make it private. that seems like the best plan.

Reply:
marked private and deprecated.